index.html

```diff
@@ -0,0 +1,141 @@
+            Newtonian mechanics</p>
+          <h5 class="section-text__title">Learn more</h5>
+        </div>
+      </div>
+    </section>
+
+    <section class="team-section">
+      <div class="section-text">
+        <h4 class="section-text__title">Team</h4>
+        <h2 class="section-text__heading">Our team</h2>
+        <p class="section-text__description">Problems trying to resolve the
+          conflict between
+          the two major realms of Classical physics: Newtonian mechanics</p>
+      </div>
+      <div class="team-section__cards-container">
+        <div class="person-card">
+          <div><img class="person-card__image" src="./images/user-cover-1.png"
+              alt="a female nurse in sky blue uniforms"></div>
+          <div class="person-card__content">
+            <h5 class="person-card__person-name">Julian Jameson</h5>
+            <h6 class="person-card__proffesion">Proffession</h6>
+            <div class="person-card__socials-container">
+              <div><img src="./images/ant-design_facebook-filled.png" alt="Facebook icon"></div>
+              <div><img src="./images/ant-design_instagram-outlined.png" alt="Instagram icon"></div>
+              <div><img src="./images/ant-design_twitter-outlined.png" alt="twitter icon"></div>
+            </div>
+          </div>
+        </div>
+        <div class="person-card">
+          <div><img class="person-card__image" src="./images/user-cover-2.png" alt="a male doctor on a red background">
+          </div>
+          <div class="person-card__content">
+            <h5 class="person-card__person-name">Julian Jameson</h5>
+            <h6 class="person-card__proffesion">Proffession</h6>
+            <div class="person-card__socials-container">
+              <div><img src="./images/ant-design_facebook-filled.png" alt="Facebook icon"></div>
+              <div><img src="./images/ant-design_instagram-outlined.png" alt="Instagram icon"></div>
+              <div><img src="./images/ant-design_twitter-outlined.png" alt="twitter icon"></div>
+            </div>
+          </div>
+        </div>
+        <div class="person-card">
+          <div><img class="person-card__image" src="./images/user-cover-3.png" alt="a female doctor with stethoscope">
+          </div>
+          <div class="person-card__content">
+            <h5 class="person-card__person-name">Julian Jameson</h5>
+            <h6 class="person-card__proffesion">Proffession</h6>
+            <div class="person-card__socials-container">
+              <div><img src="./images/ant-design_facebook-filled.png" alt="Facebook icon"></div>
+              <div><img src="./images/ant-design_instagram-outlined.png" alt="Instagram icon"></div>
+              <div><img src="./images/ant-design_twitter-outlined.png" alt="twitter icon"></div>
+            </div>
+          </div>
+        </div>
+        <div class="person-card">
+          <div><img class="person-card__image" src="./images/user-cover-4.png"
```

Comment:
If not styles applied to the div, it is not necessary. In BEM this div should have a class otherwise, you don't really need it.